apps/home/templates/apps_home/blocks/newsletter_block.html

```diff
@@ -10,92 +10,33 @@ <h2 class="title">
         </h2>
         {% endif %}
 
-        <form name="form1" id="form1" action="/newsletter-signup/" method="POST" >
-          <label for="id_email" class="form-label">{{ self.email_field_label }}<span class="icon-required" aria-hidden="true">*</span><span class="aural">({% trans 'required' %})</span></label>
-          <input class="form-control" type="text" id="id_email" name="email" required="" aria-required="true">
-
-          <div class="message message--error u-d-none" id="id_email_invalid" role="alert">{% trans 'Please fill in valid email address.' %}</div>
-          <div class="message message--success u-d-none" id="id_email_valid" role="alert">{% trans 'Your subscription to our E-Mail Newsletter was successful.' %}</div>
+        <form name="form1" id="newsletter_form" action="/newsletter-signup/" method="POST" >
```

Comment:
in order to avoid user-agent validation, maybe adding `novalidate` here?
otherwise we have double validation errors

Reply:
yes, maybe just chrome issue?

Reply:
ok from that I found another bug but good call fixed now :)